spec/acceptance/class_spec.rb

```diff
@@ -42,17 +42,21 @@ class { 'nginx':
         EOS
 
         apply_manifest(pp, catch_failures: true)
+        apply_manifest(pp, catch_changes: true)
       end
 
       describe package('nginx') do
         it { is_expected.to be_installed }
-        it 'comes from the expected source' do
-          pkg_output = shell(pkg_cmd)
-          expect(pkg_output.stdout).to match pkg_match
+        # the passenger people package ngin, except for Debian 9 systems!
+        unless fact('os.family') == 'Debian' && fact('os.release.major') == '9'
```

Comment:
I'd prefer to change ```pkg_match``` above on Debian 9 to verify it comes from Debian itself. That makes this testing a bit more straightforward.

Reply:
sounds good, I will update it.